src/applications/vaos/components/cancel/CancelAppointmentModal.jsx

```diff
@@ -42,6 +45,20 @@ export default class CancelAppointmentModal extends React.Component {
       );
     }
 
+    const isCerner = cernerFacilities?.some(facilityId =>
+      this.props.appointmentToCancel.facilityId.startsWith(facilityId),
```

Comment:
If you click on Cancel appointment for a Community Care request or appointment, do you get an error? Those appointments don't have `facilityId`.

Reply:
Yes. I believe this is resolved.